pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -321,5 +337,22 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 		"portMappings": portMappingsParam,
 	}
 
+	ingress, egress, err := bandwidth.ExtractPodBandwidthResources(annotations)
+	if err != nil {
+		return nil, fmt.Errorf("Error reading pod bandwidth annotations: %v", err)
+	}
+	if ingress != nil || egress != nil {
+		bandwidthParam := cniBandwidthEntry{}
+		if ingress != nil {
+			bandwidthParam.IngressRate = int(ingress.Value() / 1000)
+			bandwidthParam.IngressBurst = int(ingress.Value() / 1000)
```

Comment:
This value should be hard-coded to 1600 bits - it's the linux TC default and is what kubenet uses.

Reply:
Fixed. Thanks!